DeepSpeech.ipynb

```diff
@@ -1280,8 +1285,8 @@
    "source": [
     "# Define CPU as device on which the muti-gpu training is orchestrated\n",
     "with tf.device('/cpu:0'):\n",
-    "    # Obtain ted lium data\n",
-    "    ted_lium = read_data_sets(tf.get_default_graph(), './data/ted', batch_size, n_input, n_context)\n",
+    "    # Obtain all the data, defaulting to TED LIUM\n",
+    "    run_data_set = ds_importer.read_data_sets(tf.get_default_graph(), os.environ.get('ds_dataset_path', './data/ted'), batch_size, n_input, n_context)\n",
```

Comment:
nit, could you change the variable name `run_data_set` to `data_sets`to be consistent with `train`'s naming and with the fact that it's more than one data set.